src/managed/OpenLiveWriter.PostEditor/Video/VideoProvidersB2.xml

```diff
@@ -79,28 +79,28 @@
     <serviceName>YouTube</serviceName>
     <editorFormat xpp:if="version lt 15.4.3104.0816">
```

Comment:
Should there be the duplication of editorFormat and embedFormat entries? They all seem to be describing the same formats.

Reply:
A single editorFormat entry and embedFormat entry may do the trick, without the `xpp:if` version specifier. I'm not sure what exactly this does, but I have a hunch it's to do with Adobe Flash versioning.

Reply:
When I tried to search the URI, XML namespace xpp("http://writer.live.com/xmlpp/2007") it pointed to Windows Essentials. 
It is trying to detect something's version (not sure what) and showing whether to preview normal or HD quality video, based on the version it applies the query params ```?hd=1``` same applies for size element also which is defined just below.

Yes, I tried with single editorFormat and embedFormat without the ```xpp:if``` and it works. Also, in some of the below videoproviders even though size element is used based on the version editor and embedformat is used once. Should I change it to single?

Reply:
It does simplify the code a bit by having the single editorFormat and embedFormat entries. Can always be added back later if there seems to be issues 👍